pkg/workflow/workflowrun/pod.go

```diff
@@ -75,22 +76,14 @@ func (m *PodBuilder) Prepare() error {
 	podName := fmt.Sprintf("%s-%s-%s", m.wf.Name, m.stage, strings.Replace(id.String(), "-", "", -1))
 	m.pod.ObjectMeta = metav1.ObjectMeta{
 		Name:      podName,
-		Namespace: m.wfr.Namespace,
+		Namespace: m.executionContext.Namespace,
 		Labels: map[string]string{
 			common.WorkflowLabelName: "true",
 		},
 		Annotations: map[string]string{
 			common.WorkflowRunAnnotationName: m.wfr.Name,
 			common.StageAnnotationName:       m.stage,
 		},
-		OwnerReferences: []metav1.OwnerReference{
```

Comment:
`OwnerReferences` deleted, pod will still running even if the workflowrun deleted? Should we file an issue to track it?

Reply:
Pod would be deleted by controller as before when it the pod terminated.